readme.md

```diff
@@ -82,7 +93,12 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+Panda.prototype={
+    var self = this;
```

Comment:
You'd only write something like `var self = this` inside a function, whereas `prototype` is an object.